src/palindrom-dom.js

```diff
@@ -122,14 +123,9 @@ const PalindromDOM = (() => {
             this.element.dispatchEvent(beforeEvent);
 
             if (beforeEvent.defaultPrevented) {
-                // We only throw errors on people who take matters into their own hands and use getPatchUsingHTTP (not morphUrl)
-                if (throwOnCancellation) {
-                    throw new Error(
-                        '`getPatchUsingHTTP` was aborted by cancelling `palindrom-before-redirect` event.'
-                    );
-                } else {
-                    return false;
-                }
+                throw new AbortionError(
```

Comment:
LGTM, just the name for this error could be better: https://www.merriam-webster.com/dictionary/abortion